highfidelity/hifi
# Re-add locking/visibility tools to edit.js

This PR fixes an accidental regression -- the appropriate locked and visible icons should now appear when appropriate while using edit.js

To test:
1. Open edit.js
2. In the entities list, use the icons on the left to lock some entities and make some invisible.  Try toggling.
3.  Observe that the icons on the table appear correctly.
